src/app/page.tsx

```
@@ -188,151 +188,73 @@ const Home = () => {
+              <h3 className="mt-6 text-2xl font-semibold">Seamless Collaboration</h3>
+              <p className="text-gray-400 mt-4 text-center">
+                Connect with other teams easily while ensuring data privacy and secure communications.
               </p>
             </div>
           </div>
-          <button className="bg-[#e74c3c] text-white px-8 py-4 rounded-md font-medium hover:bg-[#c0392b] transition-colors">
-            Learn More About Server Management
+          <button
+            className="mt-8 bg-[#e74c3c] text-white px-10 py-4 rounded-md font-medium hover:bg-[#c0392b] transition-colors"
+            onClick={() => window.location.href = '/download'}
+          >
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)